service/templates/projects.html

```diff
@@ -633,7 +633,49 @@ <h3>Periodic Builds</h3>
             </div>
           </div>
         </div>
+        <!-- Commands for Virtual Workstations -->
+        <div class="variables">
+          <div class="form-group">
+            <div class="col-header col-lg-6 form-control-static">
+              <h3>Virtual Workstations Commands</h3>
+              <div class="muted small">Specify a command to be run on a virtual workstation for this project, and optionally a subdirectory to run it in. If using git clone, the repo and branch will be taken from this project.</div>
```

Comment:
Could you say how the command will be run? It seems that it's run by the shell at startup, but I'm not sure

Reply:
also say, commands will be: 

- run in order entered.
- all executed within the user's persistent home directory by default.

Reply:
Updated this description, but I can tweak it after creating the CLI command.